uktrade/data-workspace-frontend
# Feat/fetch your files data

### Description of change
Basic Data fetch component as part of Your Files functionality. Connects to user's S3 bucket and retrieves files and folders, then maps them to an un-styled component as list item element tags.

All functionality is hidden behind the Your Files V2 feature flag.

### How to test

At this stage all this is doing is retrieving the files and folders from an S3 bucket.

1. Go to your local admin `/admin/` and select feature flags
2. Turn on YOUR_FILES_V2_FLAG
3. Visit `/files`
4. You should the text "/bigdata" underneath the "Files" header. This is the default folder for any new user. You may see more files/folders depending on what you have stored locally in localstack.

### Checklist
